ozpricechecker/pricefinderapp/models.py

```diff
@@ -50,6 +50,22 @@ def full_url(self):
             base_url += '/'
         return urljoin(base_url, self.prod_url)
 
+    @property
+    def latest_price(self):
+        """Get the full url for this product."""
+        product_price = ProductPrice.objects.filter(product=self).order_by('-date_time').first()
+        if product_price:
+            return product_price.price
+        return None
```

Comment:
first() already returns None so the 4 lines can become a single line.
Basically the above is checking for None and then returning None if it found None

Reply:
@ericziethen 
Here need to return only price only not entire record then how can it be done in single line?